frontend/src/components/Footer/Footer.tsx

```diff
@@ -6,22 +6,22 @@ import { IconType } from '../Icon/Icon.ts';
 function Footer(): React.ReactNode {
   return (
     <StyledFooter className="footer">
-      <footer id="footer_bar" className="footerlinks">
-        <div className="footer-item">
-          <img id="logo" src="/img/icons/logo.png" alt="Logo" />
+      <footer id="footer-bar" className="footer__links">
+        <div className="footer__item">
+          <img id="footer__logo" src="/img/icons/logo.png" alt="Logo" />
         </div>
-        <div className="center-links">
-          <a href="/" className="footer-item">
+        <div className="footer__center-links">
+          <a href="/" className="footer__item">
             GitHub
           </a>
-          <a href="/" className="footer-item">
+          <a href="/" className="footer__item">
             Contacts
           </a>
-          <a href="/" className="footer-item">
+          <a href="/" className="footer__item">
             Rights
           </a>
         </div>
-        <a href="#App" className="footer-item icons">
+        <a href="#App" className="footer__item footer__icons">
```

Comment:
<a href="#App" className="footer__item footer__icons">  the mix in BEM accepts some element with block, in that case you make mix of two elements the correct suggestion is className="footer__item icons".